behaviors/TranslateBehavior.php

```diff
@@ -92,7 +92,35 @@ public function saveAttributes($event)
         $owner = $this->owner;
         foreach ($this->translateAttributes as $attribute) {
             if ($owner->isAttributeChanged($attribute)) {
-                Language::saveMessage($owner->attributes[$attribute], $this->category);
+                if (Yii::$app->sourceLanguage !== Yii::$app->language) {
+                    $translateSource = LanguageSource::find()
+                        ->where(['message' => $owner->getOldAttribute($attribute)])
+                        ->one();
+                    if ($translateSource !== null) {
+                        if (count($translateSource->languageTranslates) > 0) {
+                            foreach ($translateSource->languageTranslates as $translate) {
+                                if ($translate->language === Yii::$app->language) {
+                                    $translate->translation = $owner->attributes[$attribute];
+                                    $translate->save();
+                                    break;
+                                }
+                            }
+                        } else {
+                            $translate = new LanguageTranslate();
+                            $translate->id = $translateSource->id;
+                            $translate->language = Yii::$app->language;
+                            $translate->translation = $owner->attributes[$attribute];
+                            $translate->save();
+                        }
+
+                        $owner->$attribute = $owner->getOldAttribute($attribute);
+                        $owner->save();
```

Comment:
This save is not needed. This event is triggered before the update, so the changes made here will be saved automatically.